client/app/pages/settings/components/AuthSettings/SAMLSettings.jsx

```diff
@@ -1,45 +1,80 @@
 import React from "react";
 import Form from "antd/lib/form";
-import Checkbox from "antd/lib/checkbox";
 import Input from "antd/lib/input";
+import Radio from "antd/lib/radio";
 import DynamicComponent from "@/components/DynamicComponent";
 import { SettingsEditorPropTypes, SettingsEditorDefaultProps } from "../prop-types";
 
 export default function SAMLSettings(props) {
   const { values, onChange } = props;
 
+  const onChangeEnabledStatus = e => {
+    const updates = { auth_saml_enabled: !!e.target.value };
+    if (e.target.value) {
+      updates.auth_saml_type = e.target.value;
+    }
+    onChange(updates);
+  };
+
   return (
     <DynamicComponent name="OrganizationSettings.SAMLSettings" {...props}>
       <h4>SAML</h4>
       <Form.Item label="SAML Enabled">
-        <Checkbox
-          name="auth_saml_enabled"
-          checked={values.auth_saml_enabled}
-          onChange={e => onChange({ auth_saml_enabled: e.target.checked })}>
-          SAML Enabled
-        </Checkbox>
+        <Radio.Group
+          onChange={onChangeEnabledStatus}
+          value={values.auth_saml_enabled && (values.auth_saml_type || "static")}>
```

Comment:
```suggestion
          value={values.auth_saml_enabled && (values.auth_saml_type || "dynamic")}>
```

Following https://github.com/christophergrant/redash/pull/3#discussion_r504969251